.gitignore

```diff
@@ -0,0 +1,13 @@
+.idea
```

Comment:
I would prefer to have a list which is more tailored towards the content of this repository. And maybe add some comments of "why". At least on the top level of the repository.

Reply:
So should I comment each `.gitignore` entry?

Reply:
done ..